### PR TITLE
SHACL cleanup - moved sh:group and sh:order statements to the Schema …

### PR DESCRIPTION
**Pull request by ralphtq**
…Supplement

### CODE REVIEW

**Comment by steveraysteveray** [comment]
Status update:
Some questions/comments:
1. Weren't you going to remove the SHACL-CUSTOMIZATIONS file? It seems to still be in the imports closure.
2. The SCHEMA_QUDT-MATHEMATICS 2.1 file still contains a "is defined by" reference to 2.0
3. The rdfs:comment for qudt:Unit has an invalid datatype (latex).

...still working through things.

**Comment by steveraysteveray** [comment]
One more:
qudt:Unit-qudt_hasDimensionVector
  rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/overlay/qudt> ;
  sh:group qudt:PropertiesGroup ;
  sh:order "50" ;
.

line should be 
  sh:order "50"^^xsd:decimal ;

**Comment by dr-shorthair** [comment]
Or 
```
sh:order 50.0 ;
```

leveraging [Turtle abbreviated syntax](https://www.w3.org/TR/turtle/#abbrev)

File: schema/SCHEMA_QUDT-v2.1.ttl [hunk 1]
```diff
@@ -1622,11 +1650,16 @@ qudt:Unit
...
+  rdfs:comment "A unit of measure, or unit, is a particular quantity value that has been chosen as a scale for measuring other quantities the same kind (more generally of equivalent dimension). For example, the meter is a quantity of length that has been rigorously defined and standardized by the BIPM (International Board of Weights and Measures). Any measurement of the length can be expressed as a number multiplied by the unit meter. More formally, the value of a physical quantity Q with respect to a unit (U) is expressed as the scalar multiple of a real number (n) and U, as  \\(Q = nU\\)."^^qudt:LatexString ;
```
Comment (steveraysteveray) [inline schema/SCHEMA_QUDT-v2.1.ttl:1653]: In May, 2021, I believe we had decided to move away from using LatexString in rdfs:comment in favor of putting our Latex in dcterms:description and qudt:expression.

Comment (steveraysteveray) [reply schema/SCHEMA_QUDT-v2.1.ttl:1653]: I suggest we move this text to be the value of dcterms:description.

Comment (ralphtq) [reply schema/SCHEMA_QUDT-v2.1.ttl:1653]: RESOLVED

**Pull request merged**